lib/_http_agent.js

```diff
@@ -496,7 +496,7 @@ Agent.prototype.keepSocketAlive = function keepSocketAlive(socket) {
       const hint = RegExpPrototypeExec(/^timeout=(\d+)/, keepAliveHint)?.[1];
 
       if (hint) {
-        const serverHintTimeout = NumberParseInt(hint) * 1000;
+        const serverHintTimeout = NumberParseInt(hint, 10) * 1000;
```

Comment:
What's the benefit of this proposed change? The default value is 10?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt#parameters

> If radix is undefined or 0, it is assumed to be 10 except when the number begins with the code unit pairs 0x or 0X, in which case a radix of 16 is assumed.

I.e. this ensures the input is always interpreted as decimal.